src/configure/resources/messages.ts

```diff
@@ -110,4 +110,5 @@ export class Messages {
     public static UnableToGetTemplateParameters = "Unable to get parameters for the selected template.";
     public static AnalyzingRepo = "Analyzing your repository";
     public static AzureLoginError = "Error in getting Azure login information, Please try again.";
+    public static AdoDifferentTenantError = " The AAD tenant of your Azure DevOps repository is different than the one you have been logged-in to VSCode. Open 'Command Pallete' and select 'Azure: Sign Out' and then Sign-In to other tenant by invoking 'Deploy to Azure' extension";
```

Comment:
How sure are we that unauthorized is due to different tenant? if not 100%, then we should word the message a bit vaguely. e.g. One reason could be that the AAD tenant of your Azure DevOps repostiory is different than ...

Reply:
This error is happening while calling "_apis/git/repositories/{repo_name}" API and the user has already cloned the repo locally. So even if he has READER privilege, he must be able to do get over this API. The only other reason I can think of is that when the user has been removed from the project. 

Yeah, I will update the text.